html/gui/js/CCR.js

```diff
@@ -128,13 +128,13 @@ XDMoD.GlobalToolbar.Profile = {
 // -------------------------------------------------
 
 XDMoD.GlobalToolbar.Dashboard = {
-    text: 'Dashboard',
+    text: 'Admin Dashboard',
     scale: 'small',
     iconCls: 'btn_dashboard',
     id: 'global-toolbar-dashboard',
-    tooltip: 'Internal Dashboard',
+    tooltip: 'Load the admin dashboard in a new window.',
     handler: function () {
-        XDMoD.TrackEvent("Portal", "Dashboard Button Clicked");
+        XDMoD.TrackEvent("Portal", "Admin Dashboard Button Clicked");
```

Comment:
Looks like I shouldn't have told you to change this since we require strings to be single quotes.

```suggestion
        XDMoD.TrackEvent('Portal', 'Admin Dashboard Button Clicked');
```